conf.yaml

```diff
@@ -1436,35 +1436,50 @@ contents:
     -
         title:      Enterprise Search
         sections:
+          -
+            title:      Enterprise Search Guide
+            prefix:     en/enterprise-search
+            index:      enterprise-search-docs/index.asciidoc
+            private:    1
+            current:    *stackcurrent
+            branches:   [ 7.7 ]
```

Comment:
@jrodewig - We should build the 7.8 branch for this book as well.

Reply:
Added with 7e38646.